template.info.txt

```diff
@@ -1,7 +1,7 @@
 base     writr
-author   Anika Henke
-email    anika@selfthinker.org
-date     2016-05-02
+author   Louis Ouellet
+email    louis_ouellet@hotmail.com
+date     2023-12-21
```

Comment:
Minor: This is obviously not the correct date. I would recommend to generally never include the date in a PR and make a separate PR for a release (which basically just updates this line). Although it's also fine to make continuous releases and let this date be slightly wrong all the time. It won't matter that much, as long as later releases have a later date.

Reply:
Well at the time of the pull request it was the right date xD. But that's ok. I can update it later.

Reply:
My main point was that it will always be the wrong date unless a PR gets merged on the same day, which happens very rarely.